attestation-service/rvps/src/store/local_json/mod.rs

```diff
@@ -0,0 +1,52 @@
+use std::{collections::HashMap, fs};
+
+use super::Store;
+use crate::ReferenceValue;
+use anyhow::Result;
+use log::warn;
+use serde::Deserialize;
+use serde_json::Value;
+
+const FILE_PATH: &str = "/opt/confidential-containers/attestation-service/reference_values.json";
+
+pub struct LocalJson {
+    file_path: String,
+}
+
+#[derive(Deserialize, Default)]
+struct Config {
+    file_path: Option<String>,
+}
+
+impl LocalJson {
+    pub fn new(config: Option<Value>) -> Result<Self> {
+        let config = match config {
+            Some(v) => serde_json::from_value(v)?,
+            None => Config::default(),
+        };
+        Ok(Self {
+            file_path: config.file_path.unwrap_or_else(|| {
+                warn!("json file storage backend use default path {FILE_PATH}");
+                FILE_PATH.into()
+            }),
+        })
+    }
+}
+
+impl Store for LocalJson {
+    fn set(&mut self, name: String, rv: ReferenceValue) -> Result<Option<ReferenceValue>> {
+        let file = fs::read(&self.file_path)?;
+        let mut dic: HashMap<String, ReferenceValue> = serde_json::from_slice(&file)?;
+        let res = dic.insert(name, rv);
+        let contents = serde_json::to_vec(&dic)?;
+        fs::write(&self.file_path, contents)?;
```

Comment:
If we want to implement write (I would not recommend it), we probably want to wrap it in mutex.

Reply:
Yes. filesystem write would really bring synchronization and mutual exclusion problem. I implement this as I want it to be fully realized, for some cases users might want to use client tool to register reference values.

We surely cannot keep the file be accessed inclusively. And I do not want to make it complexer by importing Locks, do you think it would be a good option to leave it as-is without changing, but add extra note like the following?

> This is a very simple implementation. When you need to call the write interface, you need to pay attention. Its synchronization is not very good and can have unexpected behaviors. Be sure of what you are doing and then use write.

Reply:
But surely, out of my intention of implementing fs, if you still suggest to use lock here. I'd like to implement this : )

Reply:
I wouldn't protect against manual external file access either, I think it's clear that doing that while the service is running might cause corruption.

but we wiould need a mutex for concurrent access in the service. those operations are not atomic.